server/src/main/java/com/ncc/neon/services/RunService.java

```diff
@@ -15,6 +15,14 @@
 
 @Component
 public class RunService extends ElasticSearchService<Run> {
+    private static final String TRAIN_START_TIME_FIELD = "train_start_time";
+    private static final String TRAIN_END_TIME_FIELD = "train_end_time";
+    private static final String INF_START_TIME_FIELD = "inf_start_time";
+    private static final String INF_END_TIME_FIELD = "inf_end_time";
+    private static final String STATUS_FIELD = "status";
```

Comment:
Is this the same status in ModuleService?

Reply:
Kinda.  Both indices have a field named "status".